src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -5,6 +5,7 @@ Authors: Michael Stoll
 -/
 import number_theory.legendre_symbol.zmod_char
 import field_theory.finite.basic
+import number_theory.legendre_symbol.gauss_sum
 
 /-!
 # Quadratic characters of finite fields
```

Comment:
Can you add a section "Main results" in the docstring at the beginning of the file? Thanks!

Reply:
Will do.